lib/OidIndexGenerator.h

```diff
@@ -2,6 +2,9 @@
 
 #include <stdint.h>
 
+#include <saitypes.h> // For SAL annotations
```

Comment:
use sai.h and all sai should be included by using extern "C"

Reply:
or use #include "swss/sal.h"

Reply:
swss/sal.h is what I was looking for